R/create_package.R

```diff
@@ -1,9 +1,18 @@
 #' Create an R package according to INBO requirements
+#'
+#' Creates a package template in a new folder.
+#' Use this function when you want to start a new package.
+#' DO READ `vignette("getting_started")` before you run this function.
```

Comment:
Just a suggestion to make it sound less as an attack towards the reader (you is pointing at the reader in a very direct way and is often avoided).
```suggestion
#' Please DO READ `vignette("getting_started")` before running this function.
```